crates/git_ui/src/git_panel.rs

```diff
@@ -773,21 +773,35 @@ impl GitPanel {
             })
             .collect::<HashSet<_>>();
 
-        let new_co_authors = room
-            .read(cx)
-            .remote_participants()
-            .values()
-            .filter(|participant| participant.can_write())
-            .map(|participant| participant.user.as_ref())
-            .filter_map(|user| {
-                let email = user.email.as_deref()?;
-                let name = user.name.as_deref().unwrap_or(&user.github_login);
```

Comment:
We have a `name` fetched out GH profiles, but the new version falls back to the `user.github.login` entirely, is there a reason to do so?